### ISSUE TITLE
Matkin orca distribution and abundance

### ISSUE THREAD

**Issue by mbjones** [NCEAS/goa-data#1]
Matkin; rough period 90's and 2000's.

**Comment by couture322** [comment]
Eric Ward is talking to Matkin about getting bm and abundance data in a format that he can use. JC will work with Eric to get these data onto the dataONE/KNB

**Comment by couture322** [comment]
Eric is no longer interested in these data for the PFX working group:
_"Hi Jessica, yeah - I talked to Craig about it quite a bit. I actually already have the satellite tracks for something related to my work, but they weren't really the focus of the demographic comparisons we were interested in. I think what our plan is is going to be to work on some modeling with his data, and all of the other killer whale data in the NE Pacific, in parallel with, but not part of the NCEAS effort, because the timeline may be a bit longer, etc. Thanks for checking - Eric"_

Other groups are still interested in these data (also EVOS archivng efforts) so will continue to pursue this.

**Comment by couture322** [comment]
Are the data we want pkg df35d.412.2 (un-published pkg)? This package only spans 2002-2012 even though the abstract says the EVOSTC funded the project from 1989-1991, 1993, 1995-2011...
Will follow up with Matkin about:
- publishing this package,
- adding metadata (still lacking a bunch),
- getting the earlier data

**Comment by couture322** [comment]
Emailed Matkin about getting the data that Eric is asking for. Still need to follow up with him about above package. Determined that the data we need from Matkin is mark and recapture data for pods that still needs to be transformed into density estimates, which will take some time.
